meteor-app/imports/api/parts/parts.js

```diff
@@ -0,0 +1,44 @@
+import { Mongo } from 'meteor/mongo'
+import SimpleSchema from  'simpl-schema'
+
+import { createdAt, updatedAt } from '/imports/api/schema'
```

Comment:
Please also import `RegExId`, and use that for the `_id` field like this:

`_id: RegExId,`